src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -297,6 +298,23 @@
  * }
  * </pre>
  * <p>
+ * To configure to check variables, enforce
+ * no abbreviations (essentially camel case) except for
+ * words like 'ORDER', 'TEST'.
+ * </p>
+ * <p>Configuration:</p>
+ * <pre>
+ * &lt;module name="AbbreviationAsWordInName"&gt;
+ *     &lt;property name="allowedAbbreviations" value="ORDER, TEST"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *    void getORDER_TEST() {} // OK, both abbreviations are allowed
```

Comment:
We have a rule to have all code examples to be same
And only change configs and location of violations.
Examples above has MAX_ALLOWED. Let's reuse it

Reply:
Done